app/views/spree/admin/orders/_form.html.haml

```diff
@@ -0,0 +1,68 @@
+%div.admin_order_form_fields
```

Comment:
Is this CSS class actually used anywhere?

Reply:
It's used in `spec/controllers/spree/admin/line_items_controller_spec.rb:91`, previously it was a data-hook so I figured I'd just put a class instead. Should I change the test so it looks for something else in the page ?

Reply:
You've already set the test to look for a different selector, but it doesn't include this class right? Once the deface fields are removed we don't need the data-hooks anymore, and I don't think we'll need this class here.

Reply:
It's another test : https://github.com/openfoodfoundation/openfoodnetwork/blob/3259b3ecf1dcb02e9f747f18ed6deb8a06b45169/spec/controllers/spree/admin/line_items_controller_spec.rb#L91

Reply:
Yeah, I'd remove this line and change or delete that test. It's just checking that a request for HTML contains HTML, which isn't incredibly useful.

Reply:
#sorryforbeingannoyingonyourpr

Reply:
![michael_jordan_crying](https://user-images.githubusercontent.com/28786869/48014825-d5199d80-e127-11e8-941b-e2af9d903234.jpg)